shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/API/Wallet.hs

```diff
@@ -52,6 +54,14 @@ import Shelley.Spec.Ledger.Rewards
 import Shelley.Spec.Ledger.STS.Tickn (TicknState (..))
 import Shelley.Spec.Ledger.TxData (PoolParams (..), TxOut (..))
 import Shelley.Spec.Ledger.UTxO (UTxO (..))
+import qualified Shelley.Spec.Ledger.Val as Val
+
+-- | Calculate the current total stake.
```

Comment:
Maybe it might be useful to reference the spot where this is defined as "total". E.g. it's not obvious why reserves should be excluded, but treasury not.

Reply:
Good point! The reason I'm using this here is that that's also what we're using in the reward calculation, in `createRUpd`. So yes, either a comment, or maybe it's even better to have a single function calculate the total stake, and use that everywhere (this endpoint, reward calculation, and the non myopic rewards api endpoint).

Reply:
It's called "circulation" in the `createRUpd` function, perhaps that's a name we can standardise on